### PR TITLE
Beverage

### PR DESCRIPTION
**Pull request by ingkharatj**

### CODE REVIEW

File: menu.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-| Item                                   | Price |
-|:---------------------------------------|------:|
-| nothing yet                            |  0.0  |
-
-
----
-
-We accept PromptPay, KUPay, Alipay, and cash. No credit cards.
\ No newline at end of file
+| Description                | Price (Bt) |
```
Comment (ChatchapongC) [inline menu.md]: Price(Baht)

**Comment by Parkkung** [comment]
look too cheap for me

**Comment by mark47546** [comment]
absolutely nice

**Comment by bbestthi** [comment]
good

**Pull request merged**